website/server/controllers/api-v3/auth.js

```diff
@@ -227,6 +227,8 @@ api.loginLocal = {
   },
 };
 
+const supportedSocialNetworks = ['facebook', 'google'];
```

Comment:
Let's put this at the top of the file under the imports. Our convention is to use screaming snake case for these. Also, let's freeze it.

```js
const SUPPORTED_SOCIAL_NETWORKS = Object.freeze(['facebook', 'google']);
```